source/nodejs/adaptivecards-designer/package-lock.json

```diff
@@ -229,6 +229,11 @@
 				"acorn": "^5.0.0"
 			}
 		},
+		"adaptivecards": {
+			"version": "1.2.0",
```

Comment:
> "1.2.0", [](start = 13, length = 9)

You are continuing to use 1.2.0 here but then you end up changing the version for the package.json adaptivecards package to a different value --or is there no correlation?

Reply:
I have nothing to do with any `package-lock.json` file. They are automatically generated and I don't know how. I also don't know why this one references 1.2.